spyder_kernels/tests/test_kernelmixin.py

```diff
@@ -0,0 +1,389 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright © 2018- Spyder Kernels Contributors
+# Licensed under the terms of the MIT License
+# (see spyder_kernels/__init__.py for details)
+# -----------------------------------------------------------------------------
+
+"""
+Tests for the console kernel.
+"""
+
+# Standard library imports
+import os
+import os.path as osp
+
+# Test imports
+import pytest
+
+# Local imports
+from spyder_kernels.kernelmixin import PICKLE_PROTOCOL
+from spyder_kernels.utils.iofuncs import iofunctions
+from spyder_kernels.utils.test_utils import get_log_text
+
+# Third-party imports
+import cloudpickle
+
+# =============================================================================
+# Constants
+# =============================================================================
+FILES_PATH = os.path.dirname(os.path.realpath(__file__))
+
+
+# =============================================================================
+# Tests
+# =============================================================================
+# --- For the Variable Explorer
+def test_get_namespace_view(console_kernel, ipdb_kernel):
```

Comment:
@dalthviz, please avoid repeated code in these tests. For that I think you can create a single fixture called `create_kernel` parametrized with the two kernels we want to test: ConsoleKernel and IPdbKernel.